server/helper/validationArgenissEmail.js

```diff
@@ -0,0 +1,12 @@
+function checkArgenissFormatEmail(email){
+    var errorMessage = "";
+    const domain = "@argeniss.com";
+
+    if(!email || email == "" || email.indexOf(domain) < 0){
+        errorMessage += `${email}: Email have a format incorrect.`;
+    }
+
+    return errorMessage;
```

Comment:
Just return if the string is valid by matching the regex. Then the endpoints that uses this should be responsible for sending a message and error code back to the client.